profiler/src/ProfilerEngine/Datadog.Profiler.Native/FrameStore.cpp

```diff
@@ -146,7 +146,12 @@ std::pair<std::string, std::string> FrameStore::GetManagedFrame(FunctionID funct
         // try to get the type description
         if (!GetTypeDesc(pMetadataImport.Get(), classId, moduleId, mdTokenType, typeDesc))
         {
-            return {UnknownManagedAssembly, UnknownManagedType + " |fn:" + methodName};
+            // This should never happen but in case it happens, we cached the module/frame value.
```

Comment:
I know it's just a small typo, but in this context it actually changes the meaning quite a bit
```suggestion
            // This should never happen but in case it happens, we cache the module/frame value.
```